lib/bank_account.rb

```diff
@@ -10,7 +10,7 @@ def initialize(id, pin, balance)
   end
 
   def deposit(amount)
-    
+    @balance = @balance + amount
```

Comment:
We could also do something like `@balance += amount`. But this looks good for now!